src/components/Messages/Message/index.js

```diff
@@ -43,6 +44,11 @@ const onClickTranscript = async () => {
 	await handleTranscript();
 };
 
+const canPrintTranscript = () => {
+	const { transcript, printMobileTranscript } = store.state.config.settings;
+	return transcript && isMobile() ? printMobileTranscript : true;
+};
+
```

Comment:
@AlexanderKanakis When the transcript functionality is disabled (Not mobile transcript) It returns true. So it shows the `Save Transcript` button.

You can try disabling both transcript settings.

Also please check the comment I have added in the server repo for naming confusion. I think it applies here as well